app/assets/stylesheets/components/_profile.scss

```diff
@@ -1,18 +1,77 @@
 .profile {
   padding: 20px 40px 20px 40px;
-  margin-top: 80px;
-}
+  }
 
 .parent_avatar {
   border-radius: 50%;
 }
 
-.profile-card {
+.profile-card-1 {
```

Comment:
`.profile-card-1` and `.profile-card-2` are the same, so you can just call it `.profile-card` and nest the colored-box inside

Reply:
`.profile-card-3` also